src/freenas/usr/local/bin/snmp-agent.py

```diff
@@ -441,11 +441,11 @@ def get_list_of_zvols():
     agent.start()
 
     prev_zpool_info = {}
-    last_update_at = datetime.min
+    last_update_at = datetime.now(timezone.utc)
```

Comment:
We should be using monotonic clock for this, not system clock

Reply:
+1

Reply:
agreed